7.1/Dockerfile

```diff
@@ -21,6 +20,7 @@ RUN apt-get -qq update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -qq
     php7.1-intl \
     php7.1-fpm \
     php7.1-xml \
+    php7.1-mbstring \
```

Comment:
PHPUnit seems to be running well without mbstring on version 4.8, check this travis build on master 5 days ago https://travis-ci.org/kibatic/symfony-docker/jobs/261135155